Source/Scene/Cesium3DTilesetTraversal.js

```diff
@@ -1,856 +1,673 @@
 define([
         '../Core/CullingVolume',
+        '../Core/defaultValue',
```

Comment:
How is code coverage for this file?

Reply:
97%. But there are some important lines that are not being hit so I'll tweak the tests.

Reply:
Added some more tests to round out the coverage.